frontend/annotation_resolver.py

```diff
@@ -65,27 +69,27 @@ def resolve(self, annotation):
         if annotation[:4] == "List":
```

Comment:
How robust is this if there is e.g. additional whitespace in the annotation (List [ int] for example)? Might be safer to use a regex  instead.